types/index.d.ts

```diff
@@ -430,7 +430,7 @@ declare namespace Realm {
         function getAllSyncSessions(user: Realm.User): [Realm.App.Sync.Session];
         function getSyncSession(user: Realm.User, partitionValue: string|number|ObjectId|null) : Realm.App.Sync.Session;
         function setLogLevel(app: App, logLevel: LogLevel): void;
-        function setLogger(app: App, callback: (level: NumericLogLevel, message: string) => void): void;
+        function setLogger(app: App, callback: (level: string, message: string) => void): void;
```

Comment:
The API doc in `docs/sync.js` requires an update too.

Reply:
I think I make that amendment on [line 213](https://github.com/realm/realm-js/blob/022f5c2c49157e349320fd713de2eea01f11b655/docs/sync.js#L213-L214), if I'm not mistaken.